packages/core/package.js

```diff
@@ -20,27 +16,91 @@ const read = dir =>
+  return Promise.all([
+    cpy(['**/package.json'], '../dist/core/', { cwd: './src', parents: true }),
+    cpy(['./package.json'], './dist/core/', { cwd: './', parents: true }),
+    cpy(['./README.md'], './dist/core/', { cwd: './', parents: true }),
+  ]);
+}
+
+function removeCacheFiles() {
+  del.sync(
+    [
+      './dist/core/**/*.{tsbuildinfo,spec.js,spec.js.map,spec.d.ts}',
+      './dist/core/*.{tsbuildinfo,spec.js,spec.js.map,spec.d.ts}',
+    ],
+    { force: true }
+  );
+}
+
+function cleanPackageFiles() {
+  // https://open-wc.org/publishing
+  // https://justinfagnani.com/2019/11/01/how-to-publish-web-components-to-npm
+
+  read('./dist/core')
+    .filter(f => f.includes('package.json'))
+    .forEach(file => {
+      const packageFile = fs.readJsonSync(file);
+      // Angular CLI leaves in referenced dependencies
+      // https://github.com/ng-packagr/ng-packagr/pull/1372
+      // https://github.com/ng-packagr/ng-packagr/issues/1318
+      // https://github.com/angular/angular/issues/33395
+      ['__processed_by_ivy_ngcc__', 'scripts', 'devDependencies'].forEach(p => delete packageFile[p]);
+
+      const metaData = {
+        main: './index.js',
+        module: './index.js',
+        typings: './index.d.ts',
+        type: 'module',
+      };
+
+      fs.writeJsonSync(file, { ...packageFile, ...metaData }, { spaces: 2 });
+    });
+}
+
+function updateFileVersions() {
+  shell(`replace '@VERSION' $npm_package_version ./dist/core/internal/utils/global.js`);
+}
+
+function generateAPIMetaData() {
+  // We link/unlink the package so wca can properly follow the root import paths
+  shell(`cd dist/core && yarn link && yarn link @clr/core`);
+  shell(`wca analyze './dist/core/**/*.ts' --silent --format=json --outFile ./dist/core/custom-elements.json`);
+  shell(`cd dist/core && yarn unlink @clr/core && yarn unlink --no-save @clr/core`);
+
+  // update empty default slot names to have name 'default'
+  const metadata = fs.readJsonSync('./dist/core/custom-elements.json');
+  metadata.tags.filter(t => t.slots && t.slots[0].name === '').forEach(t => (t.slots[0].name = 'default'));
+  fs.writeJsonSync('./dist/core/custom-elements.json', metadata, { spaces: 2 });
+}
+
+function distributeBuild() {
+  return Promise.all([
+    // make copy into root dist for publishing
+    cpy(['./core'], '../../../dist', { cwd: './dist', parents: true }),
+
+    // copy latest for demo apps (many tools don't support symlinks so this is a workaround of sorts)
+    ...[
+      'core-angular-cli',
+      'core-angular-universal',
+      'core-create-react-app',
+      'core-ie',
+      'core-parcel-js',
+      'core-snowpack',
+      'core-vue-cli',
+    ].map(app => cpy(['./core'], `../../../apps/${app}/node_modules/@clr`, { cwd: './dist', parents: true })),
+  ]);
+}
+
+(async () => {
+  await copyAssets();
+  removeCacheFiles();
+  cleanPackageFiles();
+  updateFileVersions();
+  generateAPIMetaData();
+  await distributeBuild();
+})();
```

Comment:
Consolidated all the core packaging scripts into the single package.js. Fixes a metadata issue with Storybook not getting default slots as well as copies the dist to the demo apps for easier testing after the build has completed.